### PR TITLE
[DOC] Add service graph metrics queries

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Adds content written by @joe-elliott for service graph metrics queries. Restructures the service graph content to make pages easier to find. 

**Which issue(s) this PR fixes**:
Fixes #https://github.com/grafana/tempo-squad/issues/291

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+If you'd like to only see when a single service is the client:
+
+```promql
+sum(increase(traces_service_graph_request_server_seconds_count{client="foo"}[7d])) by (server) > 0
+```
+
+
+In all of the above queries, you can adjust the interval to change the amount of time this is calculated for. So if you wanted the same analysis done over 1d:
+
+
```
Comment (knylander-grafana) [inline docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:48]:
```suggestion

In all of the above queries, you can adjust the interval to change the amount of time this is calculated for. So if you wanted the same analysis done over 1d:

```

File: docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+A collection of useful PromQL queries for service graphs.
+
+In most cases, users want to see a visual representation of their service graph. Grafana uses the service graph metrics created by Tempo and builds that visual for the user. However, in some cases, users may want to interact with the metrics that define that service graph directly. They may want to, for example, programmatically analyze how their services are interconnected and build downstream applications that use this information. 
+
+To help with this, we've provided a collection of useful PromQL queries that can be used to explore service graph metrics. 
+
+## Instant Queries
+
+An instant query will give a single value at the end of the selected time range.
+[Instant queries](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries) are quicker to execute and easier to parse the results. We will prefer them in some scenarios:
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:19]:
```suggestion
[Instant queries](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries) are quicker to execute and it often easier to understand their results. We will prefer them in some scenarios:
```

File: docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+sum(increase(traces_service_graph_request_server_seconds_count{server="foo"}[7d])) by (client) > 0
+```
+
+If you'd like to only see when a single service is the client:
+
+```promql
+sum(increase(traces_service_graph_request_server_seconds_count{client="foo"}[7d])) by (server) > 0
+```
+
+In all of the above queries, you can adjust the interval to change the amount of time this is calculated for. So if you wanted the same analysis done over 1d:
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:43]:
```suggestion
In all of the above queries, you can adjust the interval to change the amount of time this is calculated for. So if you wanted the same analysis done over 1 day:
```

File: docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Taking two of the queries above, we can request the rate over time that any given service acted as the client or server:
+
+```promql
+sum(rate(traces_service_graph_request_server_seconds_count{server="foo"}[5m])) by (client) > 0
+
+sum(rate(traces_service_graph_request_server_seconds_count{client="foo"}[5m])) by (server) > 0
+```
+
+Notice that our interval dropped to 1 5m. This is so we only calculate the rate over the past 5 minutes which creates a more responsive graph.
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:65]:
```suggestion
Notice that our interval dropped to 5m. This is so we only calculate the rate over the past 5 minutes which creates a more responsive graph.
```

File: docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+### Latency percentiles over time between services
+
+The final queries will give us latency quantiles for the above rate. If we were interested in how the latency changed over time between any two services we could use these. In the following query the .9 will calculate the 90th percentile. Adjust this to get a p50, p99, etc.
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:69]:
```suggestion
These queries will give us latency quantiles for the above rate. If we were interested in how the latency changed over time between any two services we could use these. In the following query the `.9` means we're calculating the 90th percentile. Adjust this value if you want to calculate a different percentile for latency (e.g. p50, p95, p99, etc). 
```

**Review by 09jvilla — APPROVED** [review]
Made a few small suggestions but approving.

File: docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+menuTitle: Metrics queries
+description: Use PromQL queries to access metrics from service graphs
+weight: 300
+---
+
+# Service graph metrics queries
+
+A collection of useful PromQL queries for service graphs.
+
+In most cases, users want to see a visual representation of their service graph. Grafana uses the service graph metrics created by Tempo and builds that visual for the user. However, in some cases, users may want to interact with the metrics that define that service graph directly. They may want to, for example, programmatically analyze how their services are interconnected and build downstream applications that use this information.
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:12]: I did a bit of a rewrite here. Let me know what you think Kim.

Comment (knylander-grafana) [reply docs/sources/tempo/metrics-generator/service_graphs/metrics-queries.md:12]: Much improved. Thank you.

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**